cfgov/unprocessed/css/post.less

```diff
@@ -373,31 +373,6 @@
     - cfgov-post
 */
 
-.post_heading {
```

Comment:
Oh, good catch -- they're also referenced in a browser test. I'm looking into whether these classes are still necessary elsewhere & if we can substitute the item introduction template in any other places they're used...